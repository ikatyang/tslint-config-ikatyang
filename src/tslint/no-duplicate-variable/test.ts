// good

var good;

// bad

var bad1;
var bad1;

function bad2(a) {
  var a;
}
