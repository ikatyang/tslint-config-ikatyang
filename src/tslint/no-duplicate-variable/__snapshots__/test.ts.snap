// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

var good;

// bad

var bad1;
var bad1;
    ~~~~  [Duplicate variable: 'bad1']

function bad2(a) {
  var a;
      ~  [Duplicate variable: 'a']
}

`;
